Source/Shaders/GlobeFS.glsl

```diff
@@ -195,12 +195,16 @@ void main()
     vec4 finalColor = color;
 #endif
 
-
 #ifdef FOG
     const float fExposure = 2.0;
     vec3 fogColor = v_mieColor + finalColor.rgb * v_rayleighColor;
     fogColor = vec3(1.0) - exp(-fExposure * fogColor);
 
+#if defined(ENABLE_VERTEX_LIGHTING) || defined(ENABLE_DAYNIGHT_SHADING)
+    float darken = clamp(dot(normalize(czm_viewerPositionWC), normalize(czm_sunPositionWC)), 0.1, 1.0);
```

Comment:
`0.1` is a magic number that may not be ideal for all cases.  Is it possible to add a user configuration setting for this?  It won't need to change at runtime, I imagine any app that wanted to tweak it would just set their preferred value prior to initializing a viewer.  But I think app authors will want control over this minimum terrain brightness value.